### PR TITLE
Update extend install script

### PR DESCRIPTION
**Pull request by briandowns**
Update includes setting kernel parameters as well as adjusting how users are created and accounted for in CIS mode and updates the service files for CIS mode.

### CODE REVIEW

File: install.sh [hunk 1]
```diff
@@ -75,6 +75,15 @@ fi
 #     Create a user 'etcd'. If this value is set, the installation
 #     will chown the etcd data-dir to this user and update the etcd
 #     pod manifest.
+#
+#   - INSTALL_RKE2_CIS_MODE
+#     Enable all options to allow RKE2 to run in CIS mode if set to true. This 
+#     will change add an "etcd" system user and will update the following kernel
```
Comment (galal-hussein) [inline install.sh:81]: nit: typo here i think: "change add"

Comment (briandowns) [reply install.sh]: Thanks! Resolved.

File: install.sh [hunk 2]
```diff
@@ -841,19 +853,47 @@ create_user() {
...
+    USING_ETCD_USER=1
+}
+
+# update_kernel_params adjusts the necessary kernel parameters
+# to allow RKE2 to run in CIS mode.
+update_kernel_params() {
+    sysctl -w vm.panic_on_oom=0
+    sysctl -w kernel.panic=10
+    sysctl -w kernel.panic_on_oops=1
+    sysctl -w kernel.keys.root_maxbytes=25000000
```
Comment (dweomer) [inline install.sh:881]: These persist until the next reboot. Should we not attempt to drop-in (and apply) an `/etc/sysctl.d` entry?

Comment (briandowns) [reply install.sh:881]: I think that'd actually be preferred. I'll do that. Thanks for bringing that up!

**Pull request merged**